dessia_common/exports.py

```diff
@@ -327,7 +329,8 @@ def _object_matrix(self, object_):
         for attr, value in object_.__dict__.items():
             matrix.append([attr,
                            value.__class__.__name__,
-                           (self._value_to_str(value) if not isinstance(value, (list, dict, set)) else ' - ')])
+                           self._value_to_str(value)])
+                           # (self._value_to_str(value) if not isinstance(value, (list, dict, set)) else ' - ')])
```

Comment:
Is it important to keep this commented ?

Reply:
Removed